src/codewars.ts

```diff
@@ -28,7 +28,7 @@ type ProfileResponse = z.infer<typeof ProfileResponse>;
 
 // Retrieve a users language score, or overall score if language is undefined
 // Throws error if a user is not found
-export async function getUser(user: string, lang?: string): Promise<number> {
+export async function getUser(user: string, lang: string | null): Promise<number> {
```

Comment:
I am curious what the difference is?

Reply:
`lang?: string` is `lang: string | undefined` and doesn't accept `null`. Getting optional option returns `T | null`, so this change was required.